plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestOrcPageSourceMemoryTracking.java

```diff
@@ -590,7 +590,7 @@ public SourceOperator newTableScanOperator(DriverContext driverContext)
                     0,
                     new PlanNodeId("0"),
                     new PlanNodeId("0"),
-                    (session, split, table, columnHandles, dynamicFilter) -> pageSource,
+                    (catalog) -> (session, split, table, columnHandles, dynamicFilter) -> pageSource,
```

Comment:
Nit: no need for parens on lambda arguments. Same for other changes in this commit.